tests/services/test_record_service_files.py

```diff
@@ -160,7 +160,7 @@ def test_edit_delete(app, db, service, input_data, identity_simple, monkeypatch)
 
     # Edit draft (when no soft-deleted draft record exists)
     draft = service.edit(identity_simple, record.id)
-    assert_counts(buckets=4, objs=2, fileinstances=1, filedrafts=1, filerecords=1)
+    assert_counts(buckets=4, objs=1, fileinstances=1, filedrafts=1, filerecords=1)
```

Comment:
it made sense to me to decrease the file objects number since we are now syncing between buckets not creating hard file object copies